site/content/api/Event-API.md

```diff
@@ -76,4 +81,63 @@ The following sequence diagram illustrates the flow of messages involved in a *B
 
 **Message Format**
 
+**Arbitrary events from the device**
+
 See [*Telemetry API*]({{< relref "Telemetry-API.md" >}}) for definition of message format. 
+
+## Well-known event message types
+
+Hono defines *well-known* events that are of a specific *content-type*.
+
+In the following these events (currently one) are specified in detail.
+
+
+### Device command readiness notification
+
+By sending an event of this type the receiver is notified that a device is ready to receive a command.
+ 
+*Business Applications* may want to react to such an event by sending a command upstream to the device.
+For that purpose, the protocol adapter instance that received such an event opens a receiver link for the device (scoped to its tenant) that
+is used to send the command to the appropriate protocol adapter instance.
+
+Such an event is typically issued by the device itself. In future releases the protocol adapters may be able to send this
+event on behalf of devices (to support setups with devices that are not capable of sending such an event).
+
+
+***Device command readiness notification Payload***
+
+The following table provides an overview of the properties an event sender needs to set for a *device command readiness notification* event.
+
+| Name           | Mandatory | Location                 | Type      | Description |
+| :------------- | :-------: | :----------------------- | :-------- | :---------- |
+| *content-type* | yes       | *properties*             | *symbol*  | Must be set to *application/vnd.eclipse-hono-dcr-notification+json* |
+
+The body of the event request MUST consist of a single *AMQP Value* section containing a UTF-8 encoded string representation of a single JSON object having the following members:
+
+| Name                | Mandatory | Type       | Default Value | Description |
+| :-----------------  | :-------: | :--------- | :----------   | :---------- |
+| *source*            | *yes*     | *string*   |               | Value can be either the name of a protocol adapter (`hono-mqtt`, `hono-http`, `hono-kura`) or `device`.|
+| *stay-connected*    | no        | *long*     |               | In context with the `creation-time` (set for the AMQP 1.0 message by the protocol adapters) it defines the time interval in milliseconds in which a device should be available for receiving upstream data. If this property is missing or if it has the value 0, the device shall be considered as being always ready to receive a message. | 
```

Comment:
I propose to rename this property to ttd (time-til-disconnect) in analogy to ttl.

Also, FMPOV the scale should be seconds instead of milliseconds. I do not see the purpose of specifying 350 ms or 1756ms ...